NEWS.md

```diff
@@ -1,3 +1,6 @@
+## v3.2.1-SNAPSHOT
+* [MODSOURCE-115](https://issues.folio.org/browse/MODSOURCE-115) Create endpoint for retrieving "sourceRecords" using "matchedId" and generationId-mechanism.
```

Comment:
please move it to v3.1.4-SNAPSHOT section if we really need to release these changes as hotfix, otherwise to v3.2.0-SNAPSHOT section

Reply:
nope, this is not a bugfix, current development iteration is 3.2.0-SNAPSHOT